exe/kubernetes-run

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'kubernetes-deploy'
+require 'kubernetes-deploy/runner_task'
+require 'optparse'
+
+namespace = "default"
+command = nil
+ARGV.options do |opts|
+  opts.on("--namespace=NAMESPACE") { |n| namespace = n }
```

Comment:
huh, I had updated the script to handle the `context`. I guess I didn't push it. However, since they aren't optional you're right its best to just make them positional arugments instead.